relayer/app/opsender.go

```diff
@@ -30,8 +31,16 @@ type OpSender struct {
 // NewOpSender creates a new sender that uses txmgr to send transactions to the destination chain.
 func NewOpSender(ctx context.Context, chain netconf.Chain, rpcClient *ethclient.Client,
 	privateKey ecdsa.PrivateKey, chainNames map[uint64]string) (OpSender, error) {
-	cfg, err := txmgr.NewConfig(ctx, txmgr.NewCLIConfig(chain.RPCURL, txmgr.DefaultSenderFlagValues),
-		&privateKey, rpcClient)
+	// we want to query receipts every 1/3 of the block time
+	receiptQueryInterval := time.Duration(chain.BlockTime/3) * time.Millisecond
```

Comment:
`time.Duration(chain.BlockTime/3) * time.Millisecond` - this is an anti-pattern and should be avoided. If `BlockTime` was of type `time.Duration` you could just write `chain.BlockTime/3` - much more readable.
Please, don't use time manipulations like:
```go
time.Duration(<some_expression>) * time.<SomeUnit>
```